src/main/resources/META-INF/opentracing-global.properties

```diff
@@ -0,0 +1,3 @@
+version=${project.version}
```

Comment:
Seem @sjoerdtalsma 's codes have this always. Is this a company or personal style?